packages/engine/src/datastore/schema/field_spec/mod.rs

```diff
@@ -9,7 +9,7 @@ use std::{
 use crate::{
     datastore::error::{Error, Result},
     hash_types::state::AgentStateField,
-    simulation::package::name::PackageName,
+    simulation::package::{creator::PREVIOUS_INDEX_FIELD_NAME, name::PackageName},
```

Comment:
I guess the `PREVIOUS_INDEX_FIELD_NAME` constant could also be moved into this file, so it's together with `last_state_index_key`.

Reply:
Yeah that makes sense to me too